src/logging/framework.rs

```diff
@@ -0,0 +1,592 @@
+//! Log aggregation framework
+
+use std::cmp;
+
+use k8s_openapi::api::core::v1::Container;
+
+use crate::{builder::ContainerBuilder, commons::product_image_selection::ResolvedProductImage};
+
+use super::spec::{
+    AutomaticContainerLogConfig, ContainerLogConfig, ContainerLogConfigChoice, LogLevel,
+};
+
+/// Config directory used in the Vector log agent container
+const STACKABLE_CONFIG_DIR: &str = "/stackable/config";
+/// Directory which contains a subdirectory for every container which themselves contain the
+/// corresponding log files
+const STACKABLE_LOG_DIR: &str = "/stackable/log";
```

Comment:
Maybe make this public, so that it can be used in every operator instead of re-defining it?

Reply:
These are the locations where the volume with the log files is mounted into the **Vector container**. The locations in the product containers are by incident the same but could actually be located somewhere else. So I would not expose these constants because they are internals of the Vector container.